sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -176,17 +176,32 @@ typedef struct az_iot_hub_client_properties
 
 /**
  * @brief Initializes the Telemetry or C2D properties.
+ * 
+ * @note The properties functionality will not encode properties. In order to support
+ *       the following characters: 
+ *          `/ % # &`
+ *       the onus is on the user to properly url encode them (RFC 2396). Only these characters

```

Comment:
the onus is on the application to properly url encode them (RFC 2396). Only these characters